### ISSUE TITLE
SL90 and Infinity lights BLE Command Structure

### ISSUE THREAD

**Issue by taburineagle** [keefo/NeewerLite#42]
Hi @keefo - not an _issue_, more of a curiosity - someone asked about the SL90 light on the Issue tracker of my NeewerLite-Python program, so I got one and ran a Bluetooth packet logging session on it.  I was surprised to see the same basic command the app sends _is_ the same as the base vanilla Neewer light command, but with a lot of extra information prepended to it.  I also analyzed the Swift file for this project and saw a description of this new command under getSceneCommand(), but this (at the moment) only seems to work with the Scene tab (I tried NeewerLite with the SL90 and it worked, but only on the Scene tab).  Do you have any information on how this new command set is laid out other than the note listed under getSceneCommand()?  I understand the setSCESubTag and setSCEDataTag values, and the MAC address, but in the other (RGB/CCT) modes, those values differ from the 0x8B and 0x91 values that are programmed into NeewerLite.

**Comment by keefo** [comment]
This is my research note on new command 
https://github.com/keefo/NeewerLite/blob/main/Docs/research.md#cb60-rgb-command-reverse-engineering

The newer model of their light seems follow this pattern.

**Comment by taburineagle** [comment]
Ah, excellent - thank you!  I did run the packet analyzer, but didn't check into the APK yet... will definitely do that when I get a chance to.  I also noticed when analyzing the stream coming from the light, there tend to be around a half-dozen calls to and from the new series of lights when they're first pairing to the iOS Neewer app - probably querying what the current mode/HSL/RGB settings are, which is another upgrade from the old scheme they used!
